src/migrations/20240116000001-add-effectivedate_for_migrated_personnel.js

```diff
@@ -0,0 +1,91 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, _Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.sequelize.query(
+        `
+        -------------
+        -- PROCESS --
```

Comment:
Might be best to move each comment to its corresponding section. That way we don't have to scroll back up (just a suggestion).

Reply:
I put a copy of the number header throughout the logic